README.md

```diff
@@ -135,6 +135,38 @@ This will list all the alerts belonging to that incident.
                        (default "incident.id,alert,cluster.name,cluster.id,status,severity")
 ```
 
+## Oncall
+
+To view the current oncalls as per PagerDuty, use the command:
+
+```
+pdcli oncall
+```
+This will list all the Oncalls for each Escalation policy of **Platform-SRE** team by default.
+
+You can choose to see the Oncalls for all teams on PagerDuty using **All Teams Oncall** on the interactive window.
+
+To view your next Oncall schedule, use **Your Next Oncall Schedule** on the interactive window.
```

Comment:
I feel these lines aren't necessary, you can instead brief the same in the comment column of the oncall view navigation table.

Reply:
Incorporated these